src/main/java/racingcar/race/RacingScorecard.java

```diff
@@ -0,0 +1,24 @@
+package racingcar.race;
+
+import racingcar.util.CommonUtil;
+
+public class RacingScorecard {
```

Comment:
```suggestion
public class RacingScoreCard {
```
가 맞지 않을까요..?? 사소하지만 코멘트 달아봅니다:)